### PR TITLE
NISP-2302: Fixed country code 0 incorrectly triggering overseas Bug

### PR DESCRIPTION
**Pull request by nehakannaujia**
This issue is fixed. Now if the country is not specified, user will not be treated as overseas.

### CODE REVIEW

**Comment by swaistle** [comment]
Tested manually and can confirm it's fixed :white_check_mark:

File: app/uk/gov/hmrc/nisp/views/account_forecastonly.scala.html [hunk 1]
```diff
@@ -77,7 +77,7 @@
         <li>@Messages("nisp.main.inflation")</li>
         @if(statePension.pensionSharingOrder) {<li>@Messages("nisp.main.psod")</li>}
     </ul>
-    <h2 class="heading-medium">@NispMoney.pounds(statePension.amounts.forecast.weeklyAmount) @Messages("nisp.main.mostYouCanGet")</h2>
+    <h2 class="heading-medium">@NispMoney.pounds(statePension.amounts.forecast.weeklyAmount) @Messages("dd")</h2>
```
Comment (dspork) [inline app/uk/gov/hmrc/nisp/views/account_forecastonly.scala.html]: This seems like a mistake

File: test/uk/gov/hmrc/nisp/connectors/IdentityVerificationConnectorSpec.scala [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 HM Revenue & Customs
+ * Copyright 2017 HM Revenue & Customs
```
Comment (dspork) [inline test/uk/gov/hmrc/nisp/connectors/IdentityVerificationConnectorSpec.scala]: Could you rebase over the latest master changes so these header commits don't appear

File: app/uk/gov/hmrc/nisp/views/account_forecastonly.scala.html (hunk 1 above)
Comment (nehakannaujia) [inline app/uk/gov/hmrc/nisp/views/account_forecastonly.scala.html]: Yes, i think so. I will update it.

File: test/uk/gov/hmrc/nisp/connectors/IdentityVerificationConnectorSpec.scala (hunk 2 above)
Comment (nehakannaujia) [inline test/uk/gov/hmrc/nisp/connectors/IdentityVerificationConnectorSpec.scala]: Done.

**Pull request merged**